python/ray/log.py

```diff
@@ -0,0 +1,439 @@
+import datetime
```

Comment:
Just use `ray_logging.py` instead?  Any special reason why we create a new file here?

Reply:
`ray.log` is nice and short, but I'm okay if we want to use `ray._private.ray_logging`.